src/Microsoft.AspNet.Authentication/AuthenticationHandler.cs

```diff
@@ -155,14 +157,24 @@ public void GetDescriptions(DescribeSchemesContext describeContext)
             }
         }
 
+        protected Task<AuthenticationTicket> AuthenticateOnceAsync()
```

Comment:
Yeah sure, I ressurected the old name from before, but I agree its unclear, _authenticateTask works for me